packages/lit-html/src/lit-html.ts

```diff
@@ -296,6 +296,11 @@ export interface RenderOptions {
    * A DOM node before which to render content in the container.
    */
   renderBefore?: ChildNode | null;
+  /**
+   * ShadowRoot which is going to clone and import the TemplateInstance in case Scoped Registries
+   * are available.
+   */
+  shadowRoot?: ShadowRoot;
```

Comment:
I'm wondering if we should make this a more general name, like `ownerRoot`, `cloneContext`, or something like that, so that it's not tied only to this use case (and also because this is a speculative API and it's possible `ShadowRoot` might never get the `importNode` API, and then the field would become a misnomer).

Another use case I'm thinking about https://github.com/Polymer/lit-element/issues/1139, which (despite how we go about fixing that issue), pointed out that it's not really possible to e.g. render nodes into an iframe such that they are owned by that iframe. If that ever came back up, this could also solve that issue, e.g. 
```js
render(template, iframe.contentWindow.document, {ownerRoot: iframe.contentWindow.document})`)
```
🤷  wdyt @justinfagnani

Reply:
mmm, I was unaware about other use cases, that's why I named it shadowRoot, but makes sense to name it to something more general. If I had to choose between `ownerRoot` and `cloneContext` I prefer `ownerRoot` because IMHO `cloneContext` looks more like an action. I'll do the change.

Reply:
done